core/src/main/java/io/kestra/core/test/TestSuiteRunEntity.java

```diff
@@ -0,0 +1,106 @@
+package io.kestra.core.test;
+
+import io.kestra.core.models.DeletedInterface;
+import io.kestra.core.models.HasUID;
+import io.kestra.core.models.TenantInterface;
+import io.kestra.core.test.flow.UnitTestResult;
+
+import java.time.Instant;
+import java.util.List;
+
+
+public record TestSuiteRunEntity(
```

Comment:
```suggestion
public record TestSuiteRun(
```

We never use any suffix, only exception is Log and Metric which use the Entry suffix.
If you really want a suffix, better to use Entry to align with them

Reply:
Looking at the code it just seems useless as it's a wrapper around a TestSuiteRuneResult, we never use such persistence wrapper so I suggest getting rid of it.

Reply:
There is actually differences in serde between repositories and API. In addition TestSuiteUid is persisted to greatly faciliate queries in ES, otherwise I would have need a hack or to generate ugly opensearch query